blueman/plugins/applet/NMPANSupport.py

```diff
@@ -31,8 +31,7 @@ def service_connect_handler(service: Service, ok: Callable[[], None],
 
         return True
 
-    @staticmethod
```

Comment:
Why drop the ``@staticmethod``, self is not used.

Reply:
This is now an implementation of a `ServiceConnectHandler`. If I'm not mistaken, I'd have to make the method static there to be able to keep it static here. Having a static method there would deviate from all other callback interfaces and from an abstract point of view, it makes sense to provide `self` to the callback as it might want to use it.